### PR TITLE
Components for screen events

### PR DESCRIPTION
**Pull request by ArtemLutsenko**

### CODE REVIEW

File: frontend/screen/src/styles/_variables.scss [hunk 1]
```diff
@@ -5,4 +5,7 @@ $off-black: #585858;
 $light-grey: #E9F0F9;
```
Comment (javasia) [inline frontend/screen/src/styles/_variables.scss]: should we put such constants into 'constants' directory instead?

File: frontend/screen/src/components/ScreenEventList/ScreenEventItem/ScreenEventItem.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './ScreenEventItem.scss'
+
+const ScreenEventItem = (props) => {
+  return (
+    <div className={'sceenEventItem'} onClick={console.log("window.location='http://google.com'")}>
```
[the remark this answers is not in the document]
Comment (ArtemLutsenko) [reply frontend/screen/src/components/ScreenEventList/ScreenEventItem/ScreenEventItem.js]: just some test, deleted

File: frontend/screen/src/components/Header/MainLogo/MainLogo.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+import { NavLink } from 'react-router-dom'
 import {ReactComponent as Logo} from '../../../img/icons/MainLogo.svg'
 import './MainLogo.scss'
 
 const MainLogo = () => {
   return (
-    <div className='main-logo'>
-      <Logo />
-    </div>
+    <NavLink to="/services">
```
[the remark this answers is not in the document]
Comment (ArtemLutsenko) [reply frontend/screen/src/components/Header/MainLogo/MainLogo.js]: done

File: frontend/screen/src/components/ScreenEventList/ScreenEventList.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react'
```
[the remark this answers is not in the document]
Comment (ArtemLutsenko) [reply frontend/screen/src/components/ScreenEventList/ScreenEventList.js]: we decided to make react first

File: frontend/screen/src/styles/_variables.scss (hunk 1 above)
Comment (ArtemLutsenko) [reply frontend/screen/src/styles/_variables.scss]: its css constants. zozich put this file here

**Pull request merged**